conn2res/connectivity.py

```diff
@@ -141,12 +173,18 @@ def add_weights(self, w, mask='triu', order='random'):
             if order == 'random':
                 self.w[np.triu(self.w, 1) != 0] = w
 
-            elif order == 'rank':  # keep absolute rank of weights
+            elif order == 'absrank':  # keep absolute rank of weights
                 id_ = np.argsort(np.abs(w))
                 w = w[id_[::-1]]
                 id_ = np.argsort(np.abs(np.triu(self.w, 1)), axis=None)
                 self.w[np.unravel_index(id_[:-w.size-1:-1], self.w.shape)] = w
 
+            elif order == 'rank':  # keep rank of weights
+                id_ = np.argsort(w)
+                w = w[id_[::-1]]
+                id_ = np.argsort(np.triu(self.w, 1), axis=None)
+                self.w[np.unravel_index(id_[:-w.size-1:-1], self.w.shape)] = w
+
```

Comment:
In some cases it makes a big difference if we keep the rank or the absolute rank of weights, so I added this new option

Reply:
Great! Thank you :)